extensions/positron-python/src/client/positron/interpreterSettings.ts

```diff
@@ -0,0 +1,86 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import path from 'path';
+
+import { traceInfo, traceVerbose } from '../logging';
+import { getConfiguration } from '../common/vscodeApis/workspaceApis';
+import { arePathsSame, isParentPath } from '../pythonEnvironments/common/externalDependencies';
+import { INTERPRETERS_EXCLUDE_SETTING_KEY, INTERPRETERS_INCLUDE_SETTING_KEY } from '../common/constants';
+
+/**
+ * Gets the list of interpreters that the user has explicitly included in the settings. Converts
+ * relative and aliased paths to absolute paths.
+ * @returns List of interpreters that the user has explicitly included in the settings.
```

Comment:
Where are we converting relative and aliases paths? It looks like we're just ignoring them.

Reply:
oops good catch! I meant to resolve aliased paths and not include relative paths. I just pushed a commit to resolve aliased paths.

For relative paths, my thinking is that it's ambiguous where the paths are relative to. Are they relative to the home directory or to the project directory? Does it make sense for an admin to set a relative path as a default? I figured we can document that only absolute or absolute-but-aliased paths are valid here.